INFO:__main__:howdy! I'm going to take a look at your pump.
Traceback (most recent call last):
  File "decocare/session.py", line 123, in <module>
    stick = stick.Stick(link.Link(port, timeout=.200))
  File "/home/sharon/decoding-carelink/decocare/link.py", line 18, in __init__
    self.open( port )
  File "/home/sharon/decoding-carelink/decocare/link.py", line 27, in open
    self.serial = serial.Serial( self.port, **kwds )
  File "/usr/lib/python2.7/dist-packages/serial/serialutil.py", line 260, in __init__
    self.open()
  File "/usr/lib/python2.7/dist-packages/serial/serialposix.py", line 276, in open
    raise SerialException("could not open port %s: %s" % (self._port, msg))
serial.serialutil.SerialException: could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'
Command exited with non-zero status 1
python decocare/session.py /dev/ttyUSB0 047006
	elapsed 0:00.12
	user 0.08
	system 0.01
	CPU 81% (0text+0data 59376max)k
